index.html

```diff
@@ -560,26 +572,43 @@
         <h2>
           <a>MediaTrackSupportedConstraints</a> dictionary
         </h2>
+        <p>
+          <a>MediaTrackSupportedConstraints</a> dictionary represents the list
```

Comment:
I think it is good to repeat the text from the main spec here for completeness. Added the proposed extra text with some modifications. Also defined "applicable constraint".